examples/googlestreetview.js

```diff
@@ -14,13 +10,17 @@ goog.require('ol.style.Style');
 goog.require('ol.style.Text');
 
 goog.require('ngeo.map.module');
+goog.require('ngeo.googlestreetview.module');
+goog.require('ngeo.ToolActivate');
+goog.require('ngeo.ToolActivateMgr');
 
 
 /** @type {!angular.Module} **/
 app.module = angular.module('app', [
   ngeo.module.name,
   ngeo.map.module.name,
-  ngeo.ToolActivateMgr.module.name,
+  ngeo.googlestreetview.module.name,
+  ngeo.ToolActivateMgr.module.name
```

Comment:
For information, it is now possible to have dangling comas. It helps get cleaner diffs.

Reply:
I've learned this during this PR. Thanks